### DISCUSSION TITLE
Issues with the Snap install on Ubuntu

### DISCUSSION THREAD

**Discussion by mgrandau** [microsoft/vscode-discussions#319] [Extension Development QnA] [answered]
I noticed when I installed via the Snap service instead of the direct Deb on Ubuntu (Ubuntu 22.04.1 LTS). The extensions require install and update via VSIX.  That is fine if you're not using the nightly builds, but if you need to update regularly it is a huge pain.

I wouldn't call this a bug.  But it should be made known Through things like this forum.

**Comment by isidorn** [comment]
Can you clarify? What seems to not be working when you install via Snap? Screenshots would help.

**Comment by mgrandau** [comment]
You're forced to upgrade extensions with VSIX downloads.

**Comment by mgrandau** [comment]
I cannot reproduce anymore because I installed VSCode via Dev package. That
is the work around.

**Comment by isidorn** [comment, marked as answer]
This should just work also with the snap package. If it does not, please file a new issue against https://github.com/microsoft/vscode/issues
